src/main/java/org/zephyrsoft/sdb2/gui/MainWindow.java

```diff
@@ -2865,6 +2865,10 @@ public void setRemoteStatus(RemoteStatus status) {
 				path = "/org/zephyrsoft/sdb2/remote-orange.png";
 				tooltip = "Remote disconnecting...";
 				break;
+			case DB_DISCONNECTED:
+				path = "/org/zephyrsoft/sdb2/remote-orange.png";
+				tooltip = "Remote db offline! Please notify your admin!";
+				break;
 			case FAILURE:
 				path = "/org/zephyrsoft/sdb2/remote-red.png";
 				tooltip = "Remote connection failure! Type Strg+R to reconnect.";
```

Comment:
I know, technically this is not part of the current PR, but I noticed that it should be "Ctrl+R". I18n is saved for a later date.

Reply:
You're right. I fixed it.